bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -452,31 +457,48 @@ void doCompactEntryLogs(double threshold) {
 
         final int numBuckets = 10;
         int[] entryLogUsageBuckets = new int[numBuckets];
+        int[] compactedBuckets = new int[numBuckets];
+
+        long start = System.currentTimeMillis();
+        long end = start;
+        long timeDiff = 0;
```

Comment:
Do we need the `end` and the `timeDiff` here? Seems we can only break the following loop if (currentTime-start) >= limit?

Reply:
The timediff is held for reporting after loop exit when debugging is enabled.

Reply:
Ok  I see.